npm/webpack-dev-server/cypress/e2e/angular.cy.ts

```diff
@@ -12,83 +12,106 @@ for (const project of WEBPACK_REACT) {
     continue
   }
 
-  describe(`Working with ${project}`, () => {
+  context(project, () => {
     beforeEach(() => {
       cy.scaffoldProject(project)
       cy.openProject(project)
-      cy.startAppServer('component')
     })
 
-    it('should mount a passing test', () => {
-      cy.visitApp()
-      cy.contains('app.component.cy.ts').click()
-      cy.waitForSpecToFinish({ passCount: 1 }, 60000)
+    describe('configuration handling', () => {
+      if (!['angular-13', 'angular-14'].includes(project)) {
+        it('should initialize with unsupported browserslist entries', () => {
+          // Create .browerslistrc that requests support for ES5
+          // Support was dropped in Angular CLI v15 so this should generate a warning message in that version and beyond
+          cy.withCtx(async (ctx) => {
+            await ctx.actions.file.writeFileInProject(
+              ctx.path.resolve('.browserslistrc'),
+              'IE 11',
+            )
+          })
+
+          cy.startAppServer('component')
+          cy.visitApp()
+        })
+      }
+    })
 
-      cy.get('li.command').first().within(() => {
-        cy.get('.command-method').should('contain', 'mount')
-        cy.get('.command-message').should('contain', 'AppComponent')
+    describe('test behaviors', () => {
+      beforeEach(() => {
+        cy.startAppServer('component')
       })
-    })
 
-    it('should live-reload on src changes', () => {
-      cy.visitApp()
-      cy.contains('app.component.cy.ts').click()
-      cy.waitForSpecToFinish({ passCount: 1 }, 60000)
+      it('should mount a passing test', () => {
+        cy.visitApp()
+        cy.contains('app.component.cy.ts').click()
+        cy.waitForSpecToFinish({ passCount: 1 }, 60000)
 
-      cy.withCtx(async (ctx) => {
-        await ctx.actions.file.writeFileInProject(
-          ctx.path.join('src', 'app', 'app.component.html'),
-          (await ctx.file.readFileInProject(ctx.path.join('src', 'app', 'app.component.html'))).replace('Hello World', 'Hello Cypress'),
-        )
+        cy.get('li.command').first().within(() => {
+          cy.get('.command-method').should('contain', 'mount')
+          cy.get('.command-message').should('contain', 'AppComponent')
+        })
       })
 
-      cy.waitForSpecToFinish({ failCount: 1 }, 60000)
+      it('should live-reload on src changes', () => {
+        cy.visitApp()
+        cy.contains('app.component.cy.ts').click()
+        cy.waitForSpecToFinish({ passCount: 1 }, 60000)
```

Comment:
Not your change  but woah,60000, does the spec really take that long to execute? :turtle: